playbooks/roles/openstack-remove-compute/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Get openstack compute node id
+  shell: |
+     set -o pipefail
+     openstack compute service list | grep {{ compute_node_name }} | awk '{print $2}'
+  register: compute_id
+  delegate_to: localhost
+  failed_when: false
+
+- name: Validate the compute node id
+  fail:
+    msg: "Compute node {{ compute_node_name }} does not exists in openstack"
+  delegate_to: localhost
+  when: compute_id.stdout == ""
+
+- name: Remove label from compute node
+  command: "kubectl label node {{ compute_node_name }} {{ item }}-"
```

Comment:
this is just a question... could/what if the node name is different in kubectl, nova and ansible inventory?